plugins/catalog-react/src/components/EntityLifecyclePicker/EntityLifecyclePicker.tsx

```diff
@@ -31,6 +31,8 @@ import React, { useEffect, useMemo, useState } from 'react';
 import { useEntityListProvider } from '../../hooks/useEntityListProvider';
 import { EntityLifecycleFilter } from '../../filters';
 
+export type CatalogReactEntityLifecyclePickerClassKey = 'input';
```

Comment:
Let's add `/** @public */` doc tags to avoid the warnings

Reply:
updated